testing/e2e/workload_test.go

```diff
@@ -134,6 +196,99 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name:         "Create workload from local source and show emojis",
```

Comment:
you can remove this test. 1 test for create should suffice. This is not covering any new ground